docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-customer-specific-prices-feature-integration.md

```diff
@@ -0,0 +1,193 @@
+    {
+        return [
+            new MerchantRelationshipPriceProductCollectionDeletePlugin(),
+        ];
+    }     
+}
+```
+
+</details>
+
+<details>
+<summary markdown='span'>src/Pyz/Zed/ProductMerchantPortalGui/ProductMerchantPortalGuiDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductMerchantPortalGui;
+
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductAbstractTableConfigurationExpanderPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductConcreteTableConfigurationExpanderPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductMapperPlugin;
+use Spryker\Zed\PriceProductMerchantRelationshipMerchantPortalGui\Communication\Plugin\ProductMerchantPortalGui\MerchantRelationshipPriceProductTableFilterPlugin;
+use Spryker\Zed\ProductMerchantPortalGui\ProductMerchantPortalGuiDependencyProvider as SprykerProductMerchantPortalGuiDependencyProvider;
+
+class ProductMerchantPortalGuiDependencyProvider extends SprykerProductMerchantPortalGuiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductTableFilterPluginInterface>
+     */
+    protected function getPriceProductTableFilterPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductTableFilterPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductAbstractTableConfigurationExpanderPluginInterface>
+     */
+    protected function getPriceProductAbstractTableConfigurationExpanderPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductAbstractTableConfigurationExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductConcreteTableConfigurationExpanderPluginInterface>
+     */
+    protected function getPriceProductConcreteTableConfigurationExpanderPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductConcreteTableConfigurationExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductMerchantPortalGuiExtension\Dependency\Plugin\PriceProductMapperPluginInterface>
+     */
+    protected function getPriceProductMapperPlugins(): array
+    {
+        return [
+            new MerchantRelationshipPriceProductMapperPlugin(),
+        ];
+    }
+}
+```
+
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Log in to the Merchant Portal with a merchant with merchant relationships.
+
+Open any product for edit and make sure that the Customer column in the Price section is updatable.
```

Comment:
```suggestion
Open any merchant product and make sure that the Prices table containes "Customer" column. Make sure that you  can create a new customer price or even a butch of them by setting, amongst other things, an appropriate customer in the Customer field.  Make sure that you can delete the customer price. And make sure that you see the validation error while attempting to set or create the customer price for the volume price.```

Reply:
As for me - I'd divide this verification into several blocks per plugin group installation to discribe what each group do.